index.html

```diff
@@ -1099,7 +1099,12 @@ <h3>Workflows and Exchanges</h3>
       </p>
       <p>
 The exchange URL is given to the exchange client so that it can initiate the
-exchange. Initiating the exchange does not require any authorization beyond the
+exchange. Note that while the exchange URL is given to the coordinator to
+provide to the exchange client, the actual exchange is performed between the
+exchange client and the workflow service, not the coordinator.
```

Comment:
Just a note here that the coordinator *could* be running the exchange client -- not sure if that needs to be called out to eliminate any confusion. In other words, there's nothing wrong with a coordinator using its own exchange client to perform the exchange.

Reply:
To give an example of when a coordinator would execute the exchange it creates:

A verifier coordinator may be a website that scans barcodes that include VCBs. It then needs to verify these, which it can do by creating an exchange using an administrator-established workflow and then executing (performing the exchange) using its own exchange client. It can then display the result to the end user on the website.